src/controllers/users.js

```diff
@@ -12,6 +15,70 @@ class UsersController {
 
     return res.json(newUser);
   }
+
+  async deleteUser (req, res) {
+    try {
+      await User.deleteOne({ _id: req.params.id }).exec();
+
+      return res.status(200).json({ message: 'User is deleted.' });
+    } catch (error) {
+      res.status(400).json({ err: error });
+    }
+  }
+
+  async getUsers (req, res) {
+    const { _id, name, email, status, role, page, docs } = req.body;
```

Comment:
All these params will be in query params (req.query). Name them skip and take instead of page and docs